app/src/main/java/org/sopt/sample/util/extensions/ViewExt.kt

```diff
@@ -4,10 +4,6 @@ import android.view.View
 import com.google.android.material.snackbar.Snackbar
 
 fun View.showSnackbar(message: String, isShort: Boolean = true) {
-    val duration = if (isShort) {
-        Snackbar.LENGTH_SHORT
-    } else {
-        Snackbar.LENGTH_LONG
-    }
+    val duration = if (isShort) Snackbar.LENGTH_SHORT else Snackbar.LENGTH_LONG
     Snackbar.make(this, message, duration).show()
```

Comment:
```suggestion
    Snackbar.make(this, message, if (isShort) Snackbar.LENGTH_SHORT else Snackbar.LENGTH_LONG).show()
```
한 줄로 작성하면 깔끔해질 것 같아요~~~!

Reply:
한줄로 작성할 수도 있지만 가독성을 챙기고 싶을 때는 변수로 따로 분리하는 편,, 피드백 감사해요 :)